web/packages/design/src/Icon/README.md

```diff
@@ -2,8 +2,9 @@
 
 #### Steps:
 
-1. Download the SVG file of the icon and rename it, ensuring that it is capitalized. This is the name that will be used in the app (eg. `ArrowBack.svg`).
-2. Add the file to the `assets/` folder in this directory.
-3. Run `$ yarn process-icons`.
+1. Download the SVG file of the icon from Google Drive and rename it, ensuring that it is capitalized. This is the name that will be used in the app (eg. `ArrowBack.svg`).
+   1. PLEASE NOTE: If you are looking at icons in Figma, _please contact the design team to get an exported version_. All icons must be converted from vector lines to outlines in order to be rendered correctly in the product. Exporting an icon without preparing it will cause unpredictable rendering, such as the fill color not being updated or the entire shape being filled in with color, rather than just the outlines.
+3. Add the file to the `assets/` folder in this directory.
```

Comment:
Do we intentionally skip from step 1 to step 3 here?

Reply:
Weird. I updated this through the GitHub UI and it must have updated the numbering when I added the nested note. Interestingly, the rendered version shows the steps as 1-2-3. I'll update.